docker/rootfs/srv/gitmanager-cont-settings.py

```diff
@@ -87,4 +87,25 @@
     "REMOTE_AUTHENTICATOR_UID": "aplus",
     "DISABLE_LOGIN_CHECKS": False,
     "DISABLE_JWT_SIGNING": False,
+    "UID_TO_KEY": {
+        "aplus": """-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnME9k+VaxbUD2fGDgpKH
+Ri6cE4T6HZzqWDpvtOShhoHSYQ6VlX0YnDQYwdDoTTK+XIBG2uS8W+3CsvpxjpHF
+8Ny5xzxNGZTeqSn8A08BvoQ5cX7bnXOYUb4x2Pp/00WwaQseumUNP+ep/jCV+aqv
+iWzOmX9p8zZGdFghvplbt9A173df4t6kICK11hUm14mpUtL/bCQ2xsUEmPGX+zw8
+V1kynwJp2AaBuFVpkKDjHyHQJ+yotou01Vksp1kYoX21odjoZCivArEjuwzDEoHt
+6WHPLnwvkBYouNA9jgR63mS1rW1PiloDlNNMFW1nR+AHjTfVSKKatnswO3JVLxYe
+qwIDAQAB
+-----END PUBLIC KEY-----""",
+        "radar": """-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvddPdRsfeYjTK/aVnX/J
+52exmXiu7ZXLx2W83brLmxaOGR1Zb0qFt+0rtvBAx+s9KjlRj8rmMBLpTUTWWeai
+MpVHkhMH+MHAcL8jCfM8G5nLqcg2j50VBfXEiKT0QtonkOH+HVLYrtR0ZRQt4i9/
+8XLi1z+oITlH30yqs19PvZVksIJjReLqRDI1bxdzs6296i2Js5PyvGNKY1cn52dq
+MgMysg9P3HeuwAQW2jXQwxPn4HhNoKlQL2SpNvInWwmpS1PrXgIhEvEq+T79GcxI
+eGJ1Rjhi6HY9jhFMYBh23EirAa+HRBAbcyQ01Cc8hUa2YoNoolD/3oadsfQshps8
+8wIDAQAB
+-----END PUBLIC KEY-----""",
+     },
+     "TRUSTED_UIDS": ["radar", "aplus"],
```

Comment:
It looks like aplus-auth adds `REMOTE_AUTHENTICATOR_UID` automatically to `TRUSTED_UIDS` **only if** `TRUSTED_UIDS` is not specified at all in the settings. Therefore, we have to include `"aplus"` in `TRUSTED_UIDS` here even though it feels a bit wrong.

https://github.com/apluslms/aplus-auth/blob/8050cecd99dd9335adf684a2ec57c068e7a67edd/src/aplus_auth/__init__.py#L132-L133

However, it looks like `UID_TO_KEY` should always automatically include `REMOTE_AUTHENTICATOR_KEY` and thus, it should not be necessary to copy `"aplus"` to `UID_TO_KEY`. It has already been defined in `REMOTE_AUTHENTICATOR_KEY` and `REMOTE_AUTHENTICATOR_UID`.

https://github.com/apluslms/aplus-auth/blob/8050cecd99dd9335adf684a2ec57c068e7a67edd/src/aplus_auth/__init__.py#L135-L138

Reply:
You are correct. Removed aplus from UID_TO_KEY and force pushed.